src/main/java/org/syncninja/command/InitCommand.java

```diff
@@ -0,0 +1,21 @@
+package org.syncninja.command;
+
+import org.syncninja.model.Directory;
+import org.syncninja.service.DirectoryService;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "init")
+public class InitCommand implements Runnable {
+    private final DirectoryService directoryService = new DirectoryService();
```

Comment:
Can we make the initialization in the default constructer, it is a better practice to do that.